tests/integration/plugins/test_python_plugin.py

```diff
@@ -33,6 +34,7 @@
 
 BARE_BASES = {"bare"}
 UBUNTU_BASES = set(ALL_BASES) - BARE_BASES
+# UBUNTU_BASES = ["ubuntu:22.04"]
```

Comment:
what's this?

Reply:
leftover code that I forgot to remove
fixed